cliconfig/configfile/file.go

```diff
@@ -27,6 +27,7 @@ type ConfigFile struct {
 	PsFormat             string                      `json:"psFormat,omitempty"`
 	ImagesFormat         string                      `json:"imagesFormat,omitempty"`
 	NetworksFormat       string                      `json:"networksFormat,omitempty"`
+	PluginsFormat        string                      `json:"pluginsFormat,omitempty"`
```

Comment:
Don't forget to add an example to the example configuration file here; https://github.com/docker/docker/blob/master/docs/reference/commandline/cli.md#configuration-files